### PR TITLE
Updated Create Embed Message and Edit Message

### PR DESCRIPTION
**Pull request by RigidStudios**
**Please describe the changes this PR makes and why it should be merged:**
Changes: 
- l.215-218, no varName, doing all of that is pointless.
- l.229, the author fields don't care if they are undefined, the extra checking is unnecessary.
- l.265, removed a bunch of pointless spaghetti for a better option (Haven't decided on any default amounts, the `null`s can be replaced)
~~- l.298-323, made `Actions.createEmbed()` more advanced + docs for it.~~

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/edit_message.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ module.exports = {
 			this.callListFunc(message, "edit", [content, embed]).then(function() {
 				this.callNextAction(cache);
 			}.bind(this));
-		} else if(message && message.delete) {
+		} else if(message && message.edit && !(message.deleted)) {
```
Comment (Almeeida) [inline actions/edit_message.js:69]:
```suggestion
		} else if(message && message.edit && !message.deleted) {
```

File: actions/create_embed_message.js [hunk 2]
```diff
@@ -326,9 +293,32 @@ module.exports = {
...
+				embed.setURL(url);
+			} if (color) {
+				embed.setColor(color);
+			} if (author) {
+				embed.setAuthor(author[0], author[1], author[2]);
+			} if (description) {
+				embed.setDescription(description);
+			}
+			return embed;
 		};
```
Comment (Almeeida) [inline actions/create_embed_message.js:322]: Just get rid of this.
```suggestion
```

File: actions/create_embed_message.js [hunk 3]
```diff
@@ -259,44 +252,20 @@ module.exports = {
...
-							console.log("Invaild utc timestamp! Changed from [Custom Timestamp] to [Current Timestamp].");
-						}
-					} else {
-						embed.setTimestamp(new Date());
-						console.log("Invaild utc timestamp! from [Custom Timestamp] Changed to [Current Timestamp].");
-					}
+					embed.setTimestamp(new Date(year || null, month || null, day || null, hour || null, minute || null, second || null));
 					break;
 				default:
 					embed.setTimestamp(new Date());
```
Comment (Almeeida) [inline actions/create_embed_message.js:262]:
```suggestion
					embed.setTimestamp();
```

File: actions/create_embed_message.js [hunk 4]
```diff
@@ -259,44 +252,20 @@ module.exports = {
 						embed.setTimestamp(new Date(`${text}`));
 					} else {
 						embed.setTimestamp(new Date());
```
Comment (Almeeida) [inline actions/create_embed_message.js:254]:
```suggestion
						embed.setTimestamp();
```

File: actions/create_embed_message.js (hunk 2 above)
Comment (RigidStudios) [reply actions/create_embed_message.js:322]: Fair enough.

File: actions/create_embed_message.js (hunk 4 above)
Comment (RigidStudios) [reply actions/create_embed_message.js:254]: Honestly I disagree with this suggestion.

File: actions/create_embed_message.js (hunk 3 above)
Comment (RigidStudios) [reply actions/create_embed_message.js:262]: [here](https://github.com/dbm-network/mods/pull/397/files#r448519576)

File: actions/create_embed_message.js [hunk 5]
```diff
@@ -198,7 +198,7 @@ module.exports = {
 
 	action: function(cache) {
 		const data = cache.actions[cache.index];
-		const embed = this.createEmbed();
+		const embed = new DiscordJS.MessageEmbed();
```
Comment (Almeeida) [inline actions/create_embed_message.js:201]:
```suggestion
		const { DiscordJS } = this.getDBM();
		const embed = new DiscordJS.MessageEmbed();
```

**Pull request merged**